sdk/python/sawtooth_sdk/processor/state.py

```diff
@@ -90,3 +90,31 @@ def set(self, entries, timeout=None):
             raise InvalidTransaction(
                 'Tried to set unauthorized address: {}'.format(addresses))
         return response.addresses
+
+    def delete(self, entries, timeout=None):
+        """
+        delete an address in the validator's merkle state
+        Args:
+            entries (list): list of StateEntry
+            timeout: optional timeout, in seconds
+
+        Returns:
+            addresses (list): a list of addresses that were deleted
+
+        """
+        state_entries = [state_context_pb2.Entry(
+            address=e.address,
+            data=e.data) for e in entries]
```

Comment:
will data always be an empty string? should we force that? i.e.
`data=b'') for e in entries]` ?